### DISCUSSION TITLE
Notch on Sub-VFO problem

### DISCUSSION THREAD

**Discussion by JoeDaring** [martinbradford/101Cats#3] [General]
Hellò, Martin!

Posted this on the Groups.io Thread, but better start using this new place...

Just installed and quickly tested the last GitHub version...
Looks like most thinghs are going smooth and, also, seems that some lock-ups I was observing, at least in my set-up and when using Virtual Ports, com0com and Omnirig, with Log4OM and WSJT-X Improved, are finally gone.

One small quirk I observed is on the SUB VFO Notch... If you use the screen slider, all works fine... using the knob on the Radio makes 101Cats show a wrong value at the slider... Value seems to be: Notch Freq/10 (i.e. Notch set to 1500 Hz, value shown on screen 0150 Hz).

Just take a look at it when you can.

Thanks for all the efforts, OM.
73 de Joe, IW7ECJ

**Comment by martinbradford** [comment]
Will do, Joe...

As an experiment, can you try converting this to an Issue? Towards the bottom of the list of actions on the right hand side of this screen, there is a link "Create issue from discussion" - can you try it? I know that I can create issues, but I'm not sure if I have the permissions configured correctly for other users to be able to do so...

**Reply by JoeDaring** [reply]
Martin, looks like it worked, as I can see the issue I created from this discussion...
So... this makes me the first issue-maker on this Repo! ;-)

**Comment by martinbradford** [comment]
Got it! Will look at it tomorrow.

Martin

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by martinbradford** [reply]
ok, got it - the parameter to the CAT command is tens of Hz, not Hz. When I process the slider movement, I divide by 10 before constructing the command but when I process the incoming Auto information message, I'm not multiplying by 10.

Will fix it tomorrow...
